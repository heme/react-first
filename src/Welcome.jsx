import React, { Component } from 'react';

class Welcome extends Component {
  state = {
    welcomeMessage: 'Welcome!'
  };
  render() {
    return (
        <h1>{this.state.welcomeMessage}</h1>
    );
  }
}

export default Welcome;
