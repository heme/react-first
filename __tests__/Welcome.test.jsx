import React from 'react';
import renderer from 'react-test-renderer';
import Welcome from '../src/Welcome'

describe('Welcome', () => {

  it('renders', () => {
    const component = renderer.create(<Welcome />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });

});
