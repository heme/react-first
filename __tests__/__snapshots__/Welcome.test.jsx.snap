// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome renders 1`] = `
<h1>
  Welcome!
</h1>
`;
